src/tests/system/polarion.yaml

```diff
@@ -0,0 +1,35 @@
+testcase:
+  required:
+    title:
+      transform:
+        pattern: "^(.*)$"
+        replace: "IDM-SSSD-TC: \\1"
```

Comment:
Do we really use SSSD for authselect?

Reply:
Yes, for example, I'll have to change the work item in polarion to match the test case name, but it's currently set to. 

IDM-SSSD-TC: authselect: Authselect SSSD profile enable and disable PAM silent lastlog